alarms.json.default

```diff
@@ -207,5 +207,158 @@
 		"Dragonite":"False",
 		"Mewtwo":"False",
 		"Mew":"False"
+	},
+	"ivs":{
```

Comment:
@Spiritvn Yeah I am willing to work with you on reworking how the Pokemon are configured.